### ISSUE TITLE
Typescript support for prop style

### ISSUE THREAD

**Issue by marceloemanoel** [securingsincity/react-ace#410]
# Problem

Using react-ace with typescript forbids the user to use the `style` prop.
According to https://github.com/securingsincity/react-ace/blob/master/docs/Ace.md I should be able to pass an object to this prop. Instead, typescript yields a compilation error.
```
TS2339: Property 'style' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<AceEditor> & Readonly<{ children?: ReactNode; }> &...'.
```

## Sample code to reproduce your issue

```jsx
 <AceEditor
          readOnly
          mode="typescript"
          theme="monokai"
          value={`console.log("Hello World!");`}
          style={this.props.editorStyle}
        />
```

**Comment by agentcooper** [comment]
It seems that [`AceEditorProps`](https://github.com/securingsincity/react-ace/blob/master/types.d.ts#L125) is missing `style?: React.CSSProperties` property.

**Comment by securingsincity** [comment]
I'd appreciate a PR for this.

**Comment by dtweedle** [comment]
@securingsincity 

PR added, Plz review, it's been bugging me. Thanks a bunch :)

**Comment by securingsincity** [comment]
Thanks @dtweedle ! This is fixed in 6.1.1
